internal/pkg/aws/secretsmanager/secretsmanager.go

```diff
@@ -77,11 +78,30 @@ func (s *SecretsManager) DeleteSecret(secretName string) error {
 	})
 
 	if err != nil {
-		return fmt.Errorf("delete secret %s from secrets manager: %+v", secretName, err)
+		return fmt.Errorf("delete secret %s from secrets manager: %w", secretName, err)
 	}
 	return nil
 }
 
+// DescribeSecret retrieves the details of a secret.
+func (s *SecretsManager) DescribeSecret(secretName string) (*secretsmanager.DescribeSecretOutput, error) {
```

Comment:
can we define our own `DescribeSecretOutput` even if it's just:
```go
type DescribeSecretOutput secretsmanager.DescribeSecretOutput
```
This way clients that define an interface won't have to import both our `secretsmanager` pkg and the AWS SDK's `secretsmanager`

Reply:
Good idea! Done.